src/main/java/DukeException.java

```diff
@@ -0,0 +1,11 @@
+/**
+ * Exception class for Duke to handle Exceptions
+ */
+
+public class DukeException extends Exception {
+
+    public DukeException(String message) {
+        super(message);
+    }
```

Comment:
Awesome that this class is kept simple!

However, maybe there should be some comments to document on what kind of exceptions you are expecting? 😄